src/app.ts

```diff
@@ -265,27 +267,28 @@ class TypescriptTextWriter implements ITypescriptTextWriter {
     //this.writer.writeLine(`${name}(${parameters.map(p => p.parameter + ": " + p.type).join(", ")}): ${returnType};`);
   }
 
-  public writeLine(chunk = '') {
+  writeLine(chunk = '') {
     this.writer.writeLine(chunk);
   }
 
-  public writeNewLine(chunk = '') {
+  writeNewLine(chunk = '') {
     this.writer.writeNewLine(chunk);
   }
 
-  public write(chunk: string | TypescriptWriterCallback = '') {
+  write(chunk: string | TypescriptWriterCallback = '') {
     if (typeof chunk === 'string') {
       this.writer.write(chunk);
     } else if (typeof chunk === 'function') {
       chunk(this);
     }
   }
 
-  public end() {
+  end() {
     this.writer.end();
   }
 }
 
+// tslint:disable-next-line:no-any
 function processResource(resource: gapi.client.discovery.RestDescription): any[] {
```

Comment:
We can get rid of `any` here by replacing it with `RestMethod`:
```js
function processResource(resource: gapi.client.discovery.RestDescription): RestMethod[] {
```
and adding
```js
import RestMethod = gapi.client.discovery.RestMethod;
```
at the top of the file

Reply:
This method is also not actually used and will be removed.